docs/blog/newsletter-may-2018.rst

```diff
@@ -0,0 +1,89 @@
+
+----------------------------------
+Using imperatives in documentation
+----------------------------------
+
+English has no shortage of complexities, idiosyncrasies, and fine distinctions. Navigating them to come up with clear, concise content is at the heart of the documentarian's job. The Write the Docs Slack can be a great resource for when you need a hand with that navigation. For example, this month one documentarian asked about the difference between 'must' and 'have to', sparking a plentiful thread about not only the difference but also important considerations when using them in your docs.
+
+There is a canonical rule for their distinct usages. While both convey an obligation to the reader, "must" is for *specific* requirements, while "have to" refers to more general rules. You can read more about this definition in the `"Must vs Have to" post on Gymglish <https://www.gymglish.com/en/english-grammar/must-vs-have-to>`_.
+
+But how does that play out in the docs? A lot of it has to do with the different tone that each verb carries. 'Must' is more formal, more inescapable. It's usually a good fit for procedural content when talking about necessary actions the reader must take. 'Have to' is better for guiding your readers towards best practices or broader expectations they should be keeping in mind.
+
+*If you find yourself with other grammar or usage questions, `#general <https://writethedocs.slack.com/messages/general>`_ has many word-folks waiting to help you out.*
+
+-----------------------------------------------
+Telling the difference between one API and many
+-----------------------------------------------
+
+When writing about APIs, it's not always clear where one API stops and another begins. Some teams treat their API as singular entity that covers everything, but others treat it as a collection of APIs instead, separated by product area, function, user segment, etc. So when the question arose – How do you decide whether you're writing about one API with broad functionality or multiple individual, more targeted APIs? – the community attempted to achieve some clarity.
+
+After a thought-provoking discussion, some recurring (though not universal) definitions emerged:
+
+* Everything that lives under one hostname is one API
+* Everything that lives under one base URL and uses the same authentication specification is one API
+* Even an API with broad-ranging capabilities, loads of endpoints, and a collection of related resources is still just one API, if it's all for the same application
+
+There were some interesting examples of when teams have chosen to run contrary to these rules. For (a particularly topical) example, Facebook's Graph API and Marketing API use the same base URL, but are treated as separate APIs because they are intended for very different purposes and audiences. Similarly, Twilio's Phone and SMS APIs are treated as separate, despite using the same base URL.
```

Comment:
In the discussion, I think the person who mentioned Twilio used "Phone" and "SMS" but Twilio calls these the "Voice API" and "Messaging API" -- wondering if we should use Twilio's names (since if anyone looks it up, that's what they'll find).